### ISSUE TITLE
Warn user when tap is in use?

### ISSUE THREAD

**Issue by scanf** [imba/imba#309]
I was looking at some old Imba code and could not get the click event handlers to trigger. The issue was `:tap` would have been nice to see a warning for this.

**Comment by somebee** [comment]
I agree. Maybe we should alias `:tap` to `:click`?

**Comment by scanf** [comment]
Yeah, that would make migration easier. Also if people prefer the click, the can always rewrite. Is this something that can be done automatically by the `--migrate`?
